extensions/sql-bindings/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -53,22 +53,28 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined):
 	}));
 
 	if (!azureFunctionName) {
+		TelemetryReporter.createActionEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.exitSqlBindingsQuickpick).send();
```

Comment:
Now we're not knowing where in the flow the user is exiting though unless you piece together all the previous events. I'd suggest adding a property called like `step` or something that indicates which step the user exited the flow at.

And what's generally good practice for this to ensure we always send the exit event is to wrap the entire thing in a try/finally block and then have a `step` var updated as we go through the logic.